src/main/java/com/backend/soullive_a/entity/ModelRecentWork.java

```diff
@@ -0,0 +1,27 @@
+package com.backend.soullive_a.entity;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+@Entity
+@Getter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+
+public class ModelRecentWork {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long modelRecentWorkId;
+
+    private Integer year;
+    private String category;
```

Comment:
```suggestion
    private String category;
```
"2023 영화" 처럼 한줄에 쓰여 있어서 하나로 쓰면 좋을 것 같아요

Reply:
음 한줄에 쓰여있긴 한데, 두 개는 컬럼을 분리하는게 맞다고 생각하는데, 어떤가요?